ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/default_interface_mtu_platform.yml

```diff
@@ -9,3 +9,6 @@ default_interface_mtu: 8765 # This will not be used since platform settings take
 platform_settings:
   - platforms: [ default ]
     default_interface_mtu: 1234
+    structured_config:
+      sfe:
+        data_plane_cpu_allocation_max: 42
```

Comment:
The structured config here should not be for the platform model. It should be for the root-level device model.